sdk/keyvault/keyvault-admin/src/generated/keyVaultClientContext.ts

```diff
@@ -9,8 +9,6 @@
 import * as coreClient from "@azure/core-client";
 import { ApiVersion73, KeyVaultClientOptionalParams } from "./models";
 
-export const packageVersion = "4.3.0-beta.1";
```

Comment:
Just to clarify this - when you regen this line is _not_ emitted. It's added manually after the fact which I am trying to correct with this PR